src/main/sensors/gyro.c

```diff
@@ -159,6 +159,7 @@ typedef struct gyroSensor_s {
     gyroAnalyseState_t gyroAnalyseState;
 #endif
 
+    axis_e gyroDebugAxis;
```

Comment:
For consistency, should this be `flight_dynamics_index_t`, and `FD_ROLL` below?

Reply:
@mikeller we use the `axis_e` elements elsewhere for gyro sensor debugging - for example the gyro calibration:
https://github.com/betaflight/betaflight/blob/master/src/main/sensors/gyro.c#L852-L854

I'm in favour of consistency of course, but where is the demarcation for `flight_dynamics_index_t` v. `axis_e`? these are both in the same file.

Let me know and I will adjust.

Reply:
I agree with the idea to use roll/pitch/yaw when exposed tot he users. We have some precedence for this already with `acro_trainer_debug_axis` which accepts `ROLL | PITCH`. Also `rc_smoothing_debug_axis` although this is referring to input channels and not gyro axis. While reviewing this PR I had hoped we could unify the lookup table but unfortunately all 3 have a different valid axe list to choose from.

Reply:
Cool, agreed. I'll handle